tests/platform_tests/test_link_flap.py

```diff
@@ -85,7 +88,5 @@ def run_link_flap_test(self, dut, fanouthosts, completeness_level):
 
 @pytest.mark.platform('physical')
 def test_link_flap(request, duthost, fanouthosts):
-    normalized_level = [mark.args for mark in request.node.iter_markers(name="supported_completeness_level")][0]
-    logging.info("Completeness level set: {}".format(str(normalized_level)))
-    tlf = TestLinkFlap()
-    tlf.run_link_flap_test(duthost, fanouthosts, normalized_level)
\ No newline at end of file
+    tlf = TestLinkFlap(request)
+    tlf.run_link_flap_test(duthost, fanouthosts)
```

Comment:
Missing newline at the end. (I know you didn't add this but since we're here. 😄)

Reply:
Added.